### ISSUE TITLE
Support comments in markdown files

### ISSUE THREAD

**Issue by mkustermann** [dart-lang/markdown#114]
Github markdown seems to accept comments of the form
```
<!--  This gets ignored -->
```
It would be great if `package:markdown` could have support for that.

**Comment by srawlins** [comment]
What do you mean, "ignored"? what output are you expecting?

**Comment by srawlins** [comment]
I think we output the same as essentially all markdown implementations:

http://johnmacfarlane.net/babelmark2/?normalize=1&text=%3C!--+HTML+comments+--%3E

**Comment by mkustermann** [comment]
Sorry, here is an example of how we are using it:

```
import 'package:markdown/markdown.dart';

final String TEXT = r'''
before

<!-- ### Breaking changes -->

after
''';

main() {
  var blocks = new Document().parseLines(TEXT.split('\n'));
  print(new HtmlRenderer().render(blocks));
}
```

This results in:
```
<p>before</p>
<p>&lt;!-- ### Breaking changes --></p>
<p>after</p>
```

**Comment by srawlins** [comment]
Weird. I don't get that:

```
$ cat a.dart
import 'package:markdown/markdown.dart';

final String TEXT = r'''
before

<!-- ### Breaking changes -->

after
''';

main() {
  var blocks = new Document().parseLines(TEXT.split('\n'));
  print(new HtmlRenderer().render(blocks));
}
srawlins-macbookair:dart-markdown srawlins$ dart a.dart
<p>before</p><!-- ### Breaking changes -->
<p>after</p>
```

**Comment by mkustermann** [comment]
I bisected it now, and it turns out this was in fact fixed (namely in https://github.com/dart-lang/markdown/commit/f437d7269fc8c02c170b668d6974b863b183822c) but the lastest **published** version of `package:markdown` doesn't contain that fix.

Maybe it's just a matter of publishing a new version?

**Comment by srawlins** [comment]
You are correct! *facepalm* I didn't realize we hadn't released in forever. I just published 0.11.0. Lemme know if that works for you. Sorry!

**Comment by mkustermann** [comment]
Awesome. Thanks for publishing a new version!
